docs/tutorials/fundamentals/installation.md

```diff
@@ -230,90 +224,14 @@ page](https://github.com/napari/napari/releases) and scroll to the release you
-
-![System Preferences Security & Privacy section's General tab. At the bottom of the window there is a notification that napari was blocked and the button to "Open Anyway."](../assets/tutorials/installation/bundle_open_anyway.png)
-
-After clicking this button you'll get one final warning about napari not being
-verified, but now you'll have an "Open" button, as seen below:
+expand the `Assets` section. You can then download the appropriate file for your platform.
 
-![Finder pop up with the warning "macOS cannot verify the developer of napari. Are you sure you want to open it?" and the options "Move to Trash," "Open," or "Cancel."](../assets/tutorials/installation/bundle_open.png)
-
-After clicking "Open", the viewer should appear. Don't worry, you only have to
-go through this process once when you install a new bundle.
-
-### Installing the Windows bundle
-
-Once you have downloaded the Windows bundle zip you will have a zip file with a
-name like `napari-0.3.7-Windows.zip`. Unzip the bundle (you may like to use a
-tool like [7-zip](https://www.7-zip.org/) for this) and double click on msi
-file, eg: `napari-0.3.7.msi`
-
-The napari setup wizard will then open. Click "Next" to begin the installation.
-
-![napari Setup window with a welcome message and button to continue with "Next."](../assets/tutorials/installation/windows_bundle_installer_start.png)
-
-![napari Setup window in the process of installing napari. The progress bar has a Status that describes which step is in progress.](../assets/tutorials/installation/windows_bundle_installer_progress.png)
-
-After the setup wizard has installed napari, click "Finish" to exit.
-
-![napari Setup window success message with the button to "Finish" installation.](../assets/tutorials/installation/windows_bundle_installer_finish.png)
-
-When you launch the bundled napari app on Windows, first you'll see an empty
-command terminal appear (do not close this window - you can ignore it). The
-command terminal will be followed by a napari splash screen, and then the main
-napari user interface window will appear a few seconds later. Note that errors
-which might occur during your napari session will appear on the command terminal -
-so if something is not working, it might pay to take a look at the terminal!
-
-![Windows command terminal on napari launch.](../assets/tutorials/installation/windows_bundle_command_terminal.png)
-
-You can launch napari from the Windows start menu.
-
-![Windows start menu with napari listed as a launchable app.](../assets/tutorials/installation/windows_launch_napari.png)
-
-The very first time you launch napari the startup time will be fairly slow, but
-after that you will find that napari launches more quickly.
-
-### Installing the Linux bundle
+```{important}
+Note that the bundled app is still
+in active development, and may not be very stable. We strongly recommend
+[installing as a Python package instead](#install-as-python-package-recommended).
+```
```

Comment:
Ah! I see now. I strongly prefer the warning at the top before the user pulls the trigger, than at the bottom afterwards.

Reply:
I moved it because some parts of the documentation link to this section directly, instead of the (weird imo) forwarder above. But I don't have strong feelings; I just didn't want users to miss the warning.

Reply:
OK I see now. How about moving the warning up a bit though, to like new line 221, so after the bundle is introduced but before the directions on where to find the installers?